packages/backend/src/Controllers/Ballot/getBallotsByElectionIDController.ts

```diff
@@ -20,8 +20,20 @@ const getBallotsByElectionID = async (req: IElectionRequest, res: Response, next
         Logger.info(req, msg);
         throw new BadRequest(msg)
     }
-    Logger.debug(req, "ballots = ", ballots);
-    res.json({ election: req.election, ballots: ballots })
+
+    // Scrub identifying information from ballots to preserve voter anonymity
+    const scrubbedBallots = ballots.map(ballot => ({
```

Comment:
Looks good, this one is only used on the ballot table and it doesn't need any of those fields https://github.com/Equal-Vote/bettervoting/blob/409273b8f44e29e755d8c753cd292ce8d1c918ca/packages/frontend/src/components/Election/Admin/ViewBallots.tsx#L89